Makefile-bash.am

```diff
@@ -17,5 +17,5 @@
 # Free Software Foundation, Inc., 59 Temple Place - Suite 330,
 # Boston, MA 02111-1307, USA.
 
-completionsdir = @BASH_COMPLETIONSDIR@
+completionsdir = @bashcompletionsdir@
```

Comment:
Won't this break if `bashcompletionsdir=no`?  Presumably some other project has the correct autogoo for this?